### ISSUE TITLE
Any way to navigate in External Hard Drive Subfolders ?

### ISSUE THREAD

**Issue by DavidBabel** [GoldHEN/GoldHEN#115]
Yes we can only read pkg in the root of the connected usb HDD.

Is there any way to navigate the folders ? It's pretty hard to organize packages in the root folder.

**Comment by KrisztianKende** [comment]
Yes, we can use [PS4-Xplorer](https://pkg-zone.com/details/LAPY20009) for that.
